oci/pack.go

```diff
@@ -163,9 +163,25 @@ func NewPackageFromTarget(ctx context.Context, targ target.Target, opts ...packm
 
 	if popts.Initrd() != "" {
 		log.G(ctx).Debug("oci: including initrd")
+		initRdPath := popts.Initrd()
+		if f, err := os.Stat(initRdPath); err == nil && f.IsDir() {
+			cwd, err2 := os.Getwd()
+			if err2 != nil {
+				return nil, err2
+			}
+
+			cfg, err2 := initrd.NewFromMapping(cwd,
+				filepath.Join(filepath.Dir(initRdPath), "initramfs.cpio"),
```

Comment:
I'd turn `filepath.Dir(initRdPath), "initramfs.cpio"` into a file in tempfs, we don't want to overwrite user content potentially.